app/src/main/java/org/fossasia/openevent/general/attendees/AttendeeViewModel.kt

```diff
@@ -147,6 +150,28 @@ class AttendeeViewModel(private val attendeeService: AttendeeService, private va
                 }))
     }
 
+    fun completeOrder(charge: Charge) {
+        compositeDisposable.add(orderService.chargeOrder(orderIdentifier.toString(), charge)
+                .subscribeOn(Schedulers.io())
+                .observeOn(AndroidSchedulers.mainThread())
+                .doOnSubscribe {
+                    progress.value = true
+                }.doFinally {
+                    progress.value = false
+                }.subscribe({
+                    message.value = it.message
+                    if (it.status != null && it.status) {
+                        Timber.d("Successfully  charged for the order!")
+                    } else {
+                        Timber.d("Failed charging the user")
```

Comment:
So no feedback to user?

Reply:
Actually I am showing the feedback to the user. I am storing it in the message variable before the if else statements